dev/com.ibm.ws.concurrent.persistent/src/com/ibm/ws/concurrent/persistent/internal/PersistentExecutorIntrospector.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.concurrent.persistent.internal;
+
+import java.io.PrintWriter;
+import java.security.AccessController;
+
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.framework.ServiceReference;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+
+import com.ibm.websphere.concurrent.persistent.PersistentExecutor;
+import com.ibm.websphere.ras.annotation.Trivial;
+import com.ibm.ws.kernel.service.util.SecureAction;
+import com.ibm.wsspi.logging.Introspector;
+
+/**
+ * Introspector for persistent executors.
+ */
+@Component(configurationPolicy = ConfigurationPolicy.IGNORE)
+public class PersistentExecutorIntrospector implements Introspector {
+    @Override
+    @Trivial
+    public String getIntrospectorName() {
+        System.out.println("getIntrospectorName");
```

Comment:
Did you intend to leave System.out.println statements?

Reply:
good catch, those were just for debugging locally. I'll remove them.